web/templates/email/weekly_info.html.eex

```diff
@@ -0,0 +1,20 @@
+<h1>Thanks <%= @attendee.name %>!</h1>
+
+<p>Here is your connection card for the week of <%= @weekly_info.week_date %>:</p>
+<ul>
```

Comment:
Well, I didn't know if it would be something they may want to look back on in the future??